containres/ideaclustermanager/app/accounts/accounts_service.py

```diff
@@ -873,7 +873,19 @@ def initiate_auth(self, request: InitiateAuthRequest) -> InitiateAuthResult:
         auth_flow = request.auth_flow
         if Utils.is_empty(auth_flow):
             raise exceptions.invalid_params('auth_flow is required.')
-
+        return InitiateAuthResult(
+            challenge_name="challenge_name",
+            session="session",
+            auth={
```

Comment:
this is a total hack not mean to be any kind of production worthy anything. it's just enough to get past the login without cognito. it's a very basic jwt from jwt.io with only an added `kid` claim.

while enough to bypass auth initially, later calls which happen on the page are still failing apparently due to trying to call out to `{kid}/.well-known/jwks.json`.

more effort will need to be done to bypass the tight coupling with cognito.

we could probably setup some kind of dependency injection for the identity provider  used in the auth layer and inject something other than cognito optionally.

otherwise cognito is locked behind paywall in open stack which is something like 20 bucks per dev per month

Reply:
The good news is this does break 5 unit tests:

```
FAILED tests/unit/idea-cluster-manager/test_accounts.py::test_accounts_initiate_auth_user_password_auth_flow - AssertionError: assert 'role' == 'admin'
FAILED tests/unit/idea-cluster-manager/test_accounts.py::test_accounts_initiate_auth_user_password_auth_flow_fail - Failed: DID NOT RAISE <class 'containres.ideadatamodel.exceptions.exceptions.SocaException'>
FAILED tests/unit/idea-cluster-manager/test_accounts.py::test_accounts_initiate_auth_refresh_token_auth_flow - Failed: DID NOT RAISE <class 'containres.ideadatamodel.exceptions.exceptions.SocaException'>
FAILED tests/unit/idea-cluster-manager/test_accounts.py::test_accounts_initiate_sso_auth_flow - Failed: DID NOT RAISE <class 'containres.ideadatamodel.exceptions.exceptions.SocaException'>
FAILED tests/unit/idea-cluster-manager/test_accounts.py::test_accounts_initiate_sso_refresh_auth_flow - Failed: DID NOT RAISE <class 'containres.ideadatamodel.exceptions.exceptions.SocaException'>
```